app/api_clients/control_panel_api.js

```diff
@@ -42,31 +42,39 @@ class DjangoError extends APIError {
   constructor(error) {
     super(error);
     this.original_message = this.message;
+    const pattern = /500 - "(.*)Request Method:/;
+    const match = this.original_message.match(pattern);
+    this.message = match[1]
+      .trim()
+      .replace(/\\n/g, '<br>\n');
   }
 
   static match(error) {
     return error.message && error.message.indexOf('Traceback:') >= 0;
   }
 
   get python_traceback() {
-    let pattern = /\\nTraceback:\s+\\n\\n(.*)\\n\\nException Type/;
-    const match = this.original_message.match(pattern);
+    if (!this.original_message) {
```

Comment:
Isn't `this.original_message` always there? I may miss something.

Reply:
It could be `null` or `undefined`